### ISSUE TITLE
Package (re)naming for the latest release

### ISSUE THREAD

**Issue by gionn** [hierynomus/sshj#182]
Hi,

it looks like that in the latest release of sshj, the groupId of the artifact has been changed to *com.hierynomus*, but the library maitains the old package naming of *schmizz.net*, breaking the [1-1 correspondance between the Maven {groupId, artifactId} pair and the Java package name](http://wiki.apache.org/commons/MavenAndClasspath).

**Comment by hierynomus** [comment]
Yes, it was changed, as I took over from Shikhar as maintainer. I have push access to com.hierynomus amongst others. I didn't yet want to break all code by moving the code to a new package. But it might be indeed a good idea. Will do that with the next release.

**Comment by hierynomus** [comment]
Thought about it some more. Will not do that for the next release, but will first release a new version. The main reason is that there are a lot of fixed issues which I first want to get out, before moving all the stuff over. The next version will be `0.12.0`. When I move the classes, I will update the version of the library to `1.0.0`, notifying a (very) breaking change :wink:. Agreed?

**Comment by gionn** [comment]
As you prefer, I've already setup a dependency exclusion for the old sshj transitive dependencies I have, it isn't a blocking issue (just high chance to get punched in the face :D).

Thanks.

**Comment by exceptionfactory** [comment]
There are also several ported classes for certain algorithms that would benefit from aligned package naming.
